### PR TITLE
Add Blog page

### PR DESCRIPTION
**Pull request by mikestone14**

### CODE REVIEW

File: src/pages/Blog/Blog.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  render() {
+    return (
+      <div className={baseClass}>
+        <div className={`${baseClass}__header`}>
+          <H1SuperHeading>facebook/osquery</H1SuperHeading>
+          <Heading1>The Osquery Blog</Heading1>
+        </div>
+        {blogPosts.map((blogPost, idx) => {
+          const isLastPost = idx === blogPosts.length - 1
+          const markdown = this.converter.makeHtml(blogPost.body)
```
Comment (taylorkearns) [inline src/pages/Blog/Blog.js]: This data is markdown or html?

File: src/pages/Blog/Blog.scss [hunk 2]
```diff
@@ -0,0 +1,100 @@
+@import '../../styles/variables.scss';
+
+.blog {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+
+  &__header {
```
Comment (taylorkearns) [inline src/pages/Blog/Blog.scss]: did we want to unify on the flatter Sass style?

File: src/data/blog/index.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import frontMatter from 'front-matter'
+
+const webpackMarkdownLoader = require.context('!raw-loader!./posts', false, /\.md$/)
+const markdownFiles = webpackMarkdownLoader.keys().map(filename => {
+  const text = webpackMarkdownLoader(filename)
```
Comment (taylorkearns) [inline src/data/blog/index.js]: This section is a bit confusing to me. If I understand correctly, line 3 requires all md files in the /posts dir and concats them as `webpackMarkdownLoader`. I'm unsure where the `webpackMarkdownLoader()` method is coming from.

File: src/pages/Blog/Blog.js (hunk 1 above)
Comment (mikestone14) [reply src/pages/Blog/Blog.js]: good call, it's html

File: src/pages/Blog/Blog.scss (hunk 2 above)
Comment (mikestone14) [reply src/pages/Blog/Blog.scss]: yeah sorry, meant to change this before pushing.

File: src/data/blog/index.js (hunk 3 above)
Comment (mikestone14) [reply src/data/blog/index.js]: The way I understand it is that `require.context` creates a [context module](https://webpack.github.io/docs/context.html#context-module), and calling `keys()` on it returns an array of the filenames from the context module. Passing a filename to the context module will return the contents of that file, similar to [`require`ing it](https://webpack.github.io/docs/context.html#context-module-api).

**Pull request merged**